test/integration/simulacron-helper.js

```diff
@@ -0,0 +1,298 @@
+var helper = require('../test-helper');
+var Client = require('node-rest-client').Client;
+var spawn = require('child_process').spawn;
+
+var simulacronHelper = {
+  _execute: function(processName, params, cb) {
+    var originalProcessName = processName;
+
+    // If process hasn't completed in 10 seconds.
+    var timeout = undefined;
+    if(cb) {
+      timeout = setTimeout(function() {
+        cb("Timed out while waiting for " + processName + " to complete.");
+      }, 10000);
+    }
+
+    var p = spawn(processName, params, {});
+    p.stdout.setEncoding('utf8');
+    p.stderr.setEncoding('utf8');
+    p.stdout.on('data', function (data) {
+      console.log("%s_out> %s", originalProcessName, data);
+    });
+
+    p.stderr.on('data', function (data) {
+      console.log("%s_err> %s", originalProcessName, data);
+    });
+
+    p.on('close', function (code) {
+      console.log("%s exited with code %d", originalProcessName, code);
+      if(cb) {
+        clearTimeout(timeout);
+        if (code === 0) {
+          cb();
+        } else {
+          cb(Error("Process exited with non-zero exit code: " + code));
+        }
+      }
+    });
+
+    return p;
+  },
+  start: function(cb) {
+    var self = this;
+    var simulacronJarPath = process.env['SIMULACRON_PATH'];
```

Comment:
How about this?

- Check `SIMULACRON_PATH`, if not set, set to `$HOME/simulacron.jar`
- Check for existence of jar, if it doesn't exist, error out.

This way, if the user has their environment set up with the jar in the home directory already, there is no need to set the environment variable.  If the jar doesn't exist, an error can be emitted saying `Could not find <jarfile>, place jar in this path or set SIMULACRON_PATH appropriately`